Tasks/SonarQubePostTest/SonarQubeBuildBreaker.ps1

```diff
@@ -0,0 +1,176 @@
+. ./SonarQubeHelper.ps1
+
+$DefaultNumRetriesForAnalysisToComplete = 120; # there 1s delay between each retry plus the time it takes to make a web call
+
+#
+# Top-level orchestrating logic
+# 
+function BreakBuildOnQualityGateFailure
+{
+    
+    $breakBuild = GetTaskContextVariable "MsBuild.SonarQube.BreakBuild"            
```

Comment:
Are the task context variables case-sensitive, and if so is this the casing that is used in other tasks for MSBuild? (normal usage elsewhere is "MS", not "Ms")